src/player.cpp

```diff
@@ -10102,6 +10102,20 @@ int player::get_wind_resistance(body_part bp) const
     return totalCoverage;
 }
 
+int player::get_weight() const
```

Comment:
You probably meant `Creature` class here. `Character` has no such function, but it does have both `inventory` and the list of worn items.